test/parallel/test-event-emitter-listeners-side-effects.js

```diff
@@ -32,7 +32,6 @@ let fl;  // foo listeners
 fl = e.listeners('foo');
 assert(Array.isArray(fl));
 assert.strictEqual(fl.length, 0);
-assert(!(e._events instanceof Object));
```

Comment:
Removed this as in fast path we don't have `__proto__: null` on the events and therefor it is an object